.github/workflows/update-reproducibles.yml

```diff
@@ -0,0 +1,42 @@
+name: Update reproducible requirements
+on:
+  pull_request:
+    branches: [develop]
```

Comment:
are we going to update the PR itself, or update the develop branch, then we need to merge the PR branch from the develop branch?

Reply:
This will update the PR itself (see this commit for example in this PR: https://github.com/aws/aws-sam-cli/pull/5677/commits/32b350a69da0e7ea78b379bfc4487026a9392024)

But this GHA will only run if `requirements/base.txt` is updated (see line 6)